src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -34,7 +34,6 @@
               "file-saver",
               "lodash"
```

Comment:
@nizamial09 I know that this is not strictly related to this PR, but could you kindly add at the beginning of this list
`brace-expansion` in order to avoid this warning?
```
Warning: /ceph/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.ts depends on 'brace-expansion'. CommonJS or AMD dependencies can cause optimization bailouts.
```

Reply:
Yup. I'll do it. @alfonsomthd